assets/js/shortcuts.js

```diff
@@ -89,14 +104,18 @@ function setupShortcuts(shortcutDepth = 2) {
     if (i != 2) {
       classes += ",";
     }
-    classes += " .content-container :not([role='tabpanel']) > h" + i;
```

Comment:
Safer to leave this in?

Reply:
Yes – let's bring it back

Reply:
Ah, no, removes the shortcuts from all pages, regardless of leaf or branch bundles.

Reply:
I'm confused; why would that be? We just want to exclude header children of tabpanels. Is there something wrong with the CSS rule?